### PR TITLE
Update python dir README with getting started instructions

### LINKED ISSUE #128: Ease install of packages in our workstations
Write some documentation on how to install this package in our workstations
We should describe that you have to follow the Dockerfile to install the system libraries required

### PR DESCRIPTION
**Pull request by macrintr**
Write down the instructions I followed to build the C++ extension module so I could start developing with the python objects in this repo.

Help resolve #128.

### CODE REVIEW

File: python/README.md [hunk 1]
```diff
@@ -68,15 +68,39 @@ randomly-generated 160-bit secret key.
...
+```
+
+If you need to update to a more recent version of gcc, you can follow the
+instructions in [this guide](https://gist.github.com/application2000/73fd6f4bf1be6600a2cf9f56315a2d91).
+
+You will also need an authorization token for both Google Tasks and AWS Simple
+Tasks. The easiest way is to copy them from another workstation in lab to your
+local machine so that these two files exist in these locations:
+
+```shell
```
Comment (william-silversmith) [inline python/README.md]: I also have created the boss-secret.json, but so far that's only relevant to the BossTransferTask which is used for pipeline imports.

File: python/README.md [hunk 2]
```diff
@@ -68,15 +68,39 @@ randomly-generated 160-bit secret key.
...
 ## Development
 
-## Building the C++ extension module
+### Getting started  
 
-Mesh generation for segmentation volumes depends on a C++ extension module.  To
-build it, activate a suitable Python virtual environment and run:
+Some of the objects in this directory require mesh generation for the
+segmentation volumes, and that depends on a C++ extension module. To build it,
+you will need `gcc >= 5`. To see if your compiler is up to date, you can check
```
Comment (william-silversmith) [inline python/README.md]: I'm not sure you actually need gcc >= 5 now that I've added the changes in #140

File: python/README.md [hunk 3]
```diff
@@ -68,15 +68,39 @@ randomly-generated 160-bit secret key.
...
+
+You're ready to build the extension module. Activate a suitable Python virtual
+environment and run:
 
 ```shell
 python setup.py develop
 ```
 
+Now you're good to go. Use the [test directory](https://github.com/seung-lab/neuroglancer/tree/master/python/test) as a guide
+to write scripts.
```
Comment (william-silversmith) [inline python/README.md]: `for writing` might sound a bit better. You can also check out this directory for some examples of very dirty scripts: `https://github.com/seung-lab/neuroglancer/tree/master/python/neuroglancer/pipeline/scripts`

File: python/README.md (hunk 1 above)
Comment (macrintr) [inline python/README.md]: You can run setup.py without the Boss, so I'll leave it out.

**Pull request merged**